ee/query-service/anomaly/params.go

```diff
@@ -49,9 +56,9 @@ type anomalyQueryParams struct {
 	// and to detect anomalies
 	CurrentPeriodQuery *v3.QueryRangeParamsV3
 	// PastPeriodQuery is the query range params for past seasonal period
-	// Example: For weekly seasonality, (now-1w-4h-5m, now-1w)
```

Comment:
I forgot why did we add 4h hours last time.

Reply:
It was based on GitLab, but taking 4 hrs makes it less accurate for sudden spikes, which we want to avoid.